### ISSUE TITLE
no-wrap-func crashes on arrow expressions at start of script

### ISSUE THREAD

**Issue by btmills** [eslint/eslint#2974]
This is on current master.

```js
// test.js
/* eslint no-wrap-func: 2 */
(arg) => arg
```

```sh
$ eslint --no-eslintrc --env es6 test.js
/Users/brandon/code/eslint/eslint/lib/rules/no-wrap-func.js:53
        if (previousToken.value === "(" && nextToken.value === ")") {
                         ^
TypeError: Cannot read property 'value' of undefined
    at EventEmitter.checkFunction (/Users/brandon/code/eslint/eslint/lib/rules/no-wrap-func.js:53:26)
    at EventEmitter.emit (events.js:107:17)
    at Controller.controller.traverse.enter (/Users/brandon/code/eslint/eslint/lib/eslint.js:783:25)
    at Controller.__execute (/Users/brandon/code/eslint/eslint/node_modules/estraverse/estraverse.js:397:31)
    at Controller.traverse (/Users/brandon/code/eslint/eslint/node_modules/estraverse/estraverse.js:495:28)
    at EventEmitter.module.exports.api.verify (/Users/brandon/code/eslint/eslint/lib/eslint.js:776:24)
    at processText (/Users/brandon/code/eslint/eslint/lib/cli-engine.js:199:27)
    at processFile (/Users/brandon/code/eslint/eslint/lib/cli-engine.js:224:12)
    at /Users/brandon/code/eslint/eslint/lib/cli-engine.js:320:26
    at walk (/Users/brandon/code/eslint/eslint/lib/util/traverse.js:82:9)
```

**Comment by nzakas** [comment]
Yuck

**Comment by mysticatea** [comment]
I will try to fix this.

**Comment by btmills** [comment]
Closing this as fixed in #2982. (GitHub only auto-closes if a commit is merged to `master`, but since `no-wrap-func` is deprecated, the fix was merged to `0.x`.)
